docs/en/start/trace/skywalking.md

```diff
@@ -0,0 +1,60 @@
+# Skywalking trace 
+
+## Configuration
+
+Example: configs/config_trace_skywalking.json
+
+````json
+{
+  "tracing": {
+    "enable": true,
+    "driver": "SkyWalking",
+    "config": {
+      "reporter": "gRPC",
+      "backend_service": "127.0.0.1:11800",
+      "service_name": "layotto"
+    }
+  }
+}
+````
+
+| Field            | Required fields | Description  |
+|------------------|-----|--------------------------|
+| reporter         | Y   | Reporting method grpc               |
+| backend_service  | Y   | skywalking oap server address |
+| service_name     | Y   | Service Name                     |
+
+## Run skywalking
+
+````shell
+cd ${project_path}/diagnostics/skywalking
+
+docker-compose -f skywalking-docker-compose.yaml up -d
+````
+
+## Run layotto
+
+````shell
+cd ${project_path}/cmd/layotto_multiple_api/
+go build -o layotto
+./layotto start -c ../../configs/config_trace_skywalking.json
```

Comment:
I suggest:

~~~suggestion
```shell @if.not.exist layotto
# build it
go build -o layotto
```

Run it:

```shell @background
./layotto start -c ../../configs/config_trace_skywalking.json
```
~~~

We need to add the  annotation `@background` so that it would be run as a background job and won't block the execution